docs/index.md

```diff
@@ -0,0 +1,35 @@
+# TorchServe
+
+TorchServe is a flexible and easy to use tool for serving PyTorch models.
+
+**WARNING:** TorchServe is experimental and subject to change.
+
+
+## Basic Features
+
+* [Serving Quick Start](https://github.com/pytorch/serve/blob/master/README.md#serve-a-model) - Basic server usage tutorial
+* [Model Archive Quick Start](https://github.com/pytorch/serve/tree/master/model-archiver#creating-a-model-archive) - Tutorial that shows you how to package a model archive file.
+* [Installation](https://github.com/pytorch/serve/blob/master/README.md#install-torchserve) - Installation procedures
+* [Serving Models](server.md) - Explains how to use torchserve
+   * [REST API](rest_api.md) - Specification on the API endpoint for TorchServe
+* [Packaging Model Archive](https://github.com/pytorch/serve/tree/master/model-archiver#torch-model-archiver-for-torchserve) - Explains how to package model archive file, use `model-archiver`.
+* [Logging](logging.md) - How to configure logging
+* [Metrics](metrics.md) - How to configure metrics
+   * [Metrics API](metrics_api.md) - How to configure metrics API
+* [Batch inference with TorchServe](batch_inference_with_ts.md) - How to create and serve a model with batch inference in TorchServe
+* [Model Zoo](model_zoo.md) - List of pre-trained model archives ready to be served for inference with TorchServe.
+
+## Advanced Features
+
+* [Advanced configuration](configuration.md) - Describes advanced TorchServe configurations.
+* [Custom Service](custom_service.md) - Describes how to develop custom inference services.
+* [Unit Tests](https://github.com/pytorch/serve/tree/master/ts/tests#testing-torchserve) - Housekeeping unit tests for TorchServe.
+* [Benchmark](https://github.com/pytorch/serve/tree/master/benchmarks#torchserve-model-server-benchmarking) - Use JMeter to run TorchServe through the paces and collect benchmark data.
+* [TorchServe on Kubernetes](https://github.com/pytorch/serve/blob/master/kubernetes/README.md) -  Demonstrates a Torchserve deployment in Kubernetes using Helm Chart.
```

Comment:
This works as expected. But the other link - https://jlin27.github.io/serve-1/#advanced-features - This *TorchServe on Kubernetes* link seems to be broken. 

Is there a way to view the latest changes in HTML ?

Reply:
Hi! I've fixed the link by appending an html bookmark to the .md file link to the content that generates HTML. You can see the changes in HTML [here](https://jeremiahschung.github.io/ghactions/) where I hosted the generated content through GitHub Pages after cloning the repo. You could also try generating the HTML files locally by running through the steps here in [doc_automation.yml](https://github.com/pytorch/serve/pull/275/files) from line 16~28.